File: examples/tmp/py/create_img.py
from daytona_sdk import Daytona, Image

daytona = Daytona()

image = Image.base("python:3.13.2-slim-bookworm").run_commands("pip install uuid matplotlib seaborn numpy scipy")

print("Creating image...")
daytona.create_image("id-test-image:0.0.69", image, on_logs=print)
print("Image created")

File: examples/tmp/ts/create_img.ts
import { Daytona, Image } from '@daytonaio/sdk'

async function main() {
  const daytona = new Daytona()

  const image = Image.base('python:3.13.2-slim-bookworm').runCommands('pip install uuid pandas')
  // .pipInstallFromPyproject('packages/python/pyproject.toml', {
  //   optionalDependencies: ['dev'],
  // })
  .addLocalDir('packages/python', '/home/daytona/ws/python-pkg')
  // .addLocalFile('.devcontainer/devcontainer.json', '/home/daytona/.devcontainer/devcontainer.json')

  await daytona.createImage('test-image:0.0.101', image, { onLogs: console.log })
}

main()
